### DISCUSSION TITLE
How to get wave-preview in header?

### DISCUSSION THREAD

**Discussion by saporator** [quodlibet/quodlibet#4270] [General]
Hi, 
I'm new to Quod Libet.

When I play a mp3 I'm not able to get the "wave preview" which is  shown on the page https://quodlibet.readthedocs.io/en/latest/screenshots.html . 
What do I need to do to get this?
![grafik](https://user-images.githubusercontent.com/59700253/220895984-310d1eb7-e823-4e05-b6a4-1e5ff895a729.png)

Thank you

**Comment by jej** [comment]
This is managed by the "Waveform Seek Bar" plugin. You need to activate it. Menu File->Plugins.

![screenshot_20230223-125417](https://user-images.githubusercontent.com/33821/220898763-f83b2994-b9e9-4f5d-93c6-bdb8b80463e8.png)

**Reply by saporator** [reply]
Thank you! I haven't found it as the plugins are translated, I might switch to English.

**Reply by jej** [reply]
Just search for the English name, you should find it.
